.github/workflows/ca-existing-ds-test.yml

```diff
@@ -245,6 +245,16 @@ jobs:
           docker exec ds dsconf localhost backend suffix list | tee actual
           diff expected actual
 
+      - name: Get number of users with nsPagedSizeLimit attribute set
+        run: |
+          docker exec ds ldapsearch \
+              -H ldap://ds.example.com:3389 \
+              -D "cn=Directory Manager" \
+              -b "dc=example,dc=com" \
+              -w Secret.123 \
+              + \
+              | grep "nsPageSizeLimit: 20000" | wc -l
+
```

Comment:
Maybe, we could add test with the expected value.

Reply:
I didn't want to make it too fragile, someone may want to add additional users (or remove one) and that would break this. Actually, the `wc -l` part was a leftover from something I was trying but it is not necessary. `grep` returns 1 if there is no match and that is sufficient for us, I think. I will drop `wc -l` before merging as that is returning 0 and probably hiding the error code from `grep`.

Reply:
https://github.com/dogtagpki/pki/actions/runs/4064058832/jobs/6997310640
Turns out `wc -l` was hiding `grep`s return code of 1! Not quite sure why, perhaps I needed to make the `-b` argument the same as the other tests?

Reply:
How about something like this?
```
echo "dn: uid=pkidbuser,ou=People,dc=ca,dc=pki,dc=example,dc=com" > expected
echo "nsPagedSizeLimit: 20000" >> expected

docker exec ds ldapsearch \
    -H ldap://ds.example.com:3389 \
    -D "cn=Directory Manager" \
    -w Secret.123 \
    -b "uid=pkidbuser,ou=People,dc=ca,dc=pki,dc=example,dc=com" \
    nsPagedSizeLimit \
    | tee actual

diff expected actual
```
This way we test `pkidbuser` specifically, the expected value is clearly defined, and if there's a problem we can see the actual value in the log.

Reply:
OK, this might be what @fmarco76 was alluding to before, sorry if I misunderstood you!

Out of interest, why is the port 3389 here rather than 389? Is this a container related thing?

Reply:
I think you can add `-o ldif_wrap=no` and `-LLL` to the `ldapsearch` command to normalize and clean up the output for verification.

Reply:
Thanks, will be useful. I'll merge once I finally make this test pass :laughing: